[2024-02-21T07:37:49.228+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: extract_email.get_mail scheduled__2024-01-19T00:00:00+00:00 [queued]>
[2024-02-21T07:37:49.254+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: extract_email.get_mail scheduled__2024-01-19T00:00:00+00:00 [queued]>
[2024-02-21T07:37:49.255+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 1
[2024-02-21T07:37:49.282+0000] {taskinstance.py:2191} INFO - Executing <Task(PythonOperator): get_mail> on 2024-01-19 00:00:00+00:00
[2024-02-21T07:37:49.299+0000] {standard_task_runner.py:60} INFO - Started process 67 to run task
[2024-02-21T07:37:49.302+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'extract_email', 'get_mail', 'scheduled__2024-01-19T00:00:00+00:00', '--job-id', '57', '--raw', '--subdir', 'DAGS_FOLDER/extract_email.py', '--cfg-path', '/tmp/tmp2p6v_fw_']
[2024-02-21T07:37:49.304+0000] {standard_task_runner.py:88} INFO - Job 57: Subtask get_mail
[2024-02-21T07:37:49.484+0000] {task_command.py:423} INFO - Running <TaskInstance: extract_email.get_mail scheduled__2024-01-19T00:00:00+00:00 [running]> on host 893865381b0d
[2024-02-21T07:37:49.780+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='extract_email' AIRFLOW_CTX_TASK_ID='get_mail' AIRFLOW_CTX_EXECUTION_DATE='2024-01-19T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-01-19T00:00:00+00:00'
[2024-02-21T07:37:49.782+0000] {taskinstance.py:2698} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/taskinstance.py", line 433, in _execute_task
    result = execute_callable(context=context, **execute_callable_kwargs)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/operators/python.py", line 196, in execute
    context_merge(context, self.op_kwargs, templates_dict=self.templates_dict)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/utils/context.py", line 261, in context_merge
    context.update(*args, **kwargs)
  File "/usr/local/lib/python3.8/_collections_abc.py", line 837, in update
    for key, value in other:
ValueError: too many values to unpack (expected 2)
[2024-02-21T07:37:49.915+0000] {taskinstance.py:1138} INFO - Marking task as FAILED. dag_id=extract_email, task_id=get_mail, execution_date=20240119T000000, start_date=20240221T073749, end_date=20240221T073749
[2024-02-21T07:37:49.942+0000] {standard_task_runner.py:107} ERROR - Failed to execute job 57 for task get_mail (too many values to unpack (expected 2); 67)
[2024-02-21T07:37:49.993+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 1
[2024-02-21T07:37:50.066+0000] {taskinstance.py:3280} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-02-21T07:59:29.584+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: extract_email.get_mail scheduled__2024-01-19T00:00:00+00:00 [queued]>
[2024-02-21T07:59:29.595+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: extract_email.get_mail scheduled__2024-01-19T00:00:00+00:00 [queued]>
[2024-02-21T07:59:29.596+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 1
[2024-02-21T07:59:29.614+0000] {taskinstance.py:2191} INFO - Executing <Task(PythonOperator): get_mail> on 2024-01-19 00:00:00+00:00
[2024-02-21T07:59:29.621+0000] {standard_task_runner.py:60} INFO - Started process 344 to run task
[2024-02-21T07:59:29.625+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'extract_email', 'get_mail', 'scheduled__2024-01-19T00:00:00+00:00', '--job-id', '114', '--raw', '--subdir', 'DAGS_FOLDER/extract_email.py', '--cfg-path', '/tmp/tmpjql07a62']
[2024-02-21T07:59:29.628+0000] {standard_task_runner.py:88} INFO - Job 114: Subtask get_mail
[2024-02-21T07:59:29.681+0000] {task_command.py:423} INFO - Running <TaskInstance: extract_email.get_mail scheduled__2024-01-19T00:00:00+00:00 [running]> on host 9bab6a0db245
[2024-02-21T07:59:29.772+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='extract_email' AIRFLOW_CTX_TASK_ID='get_mail' AIRFLOW_CTX_EXECUTION_DATE='2024-01-19T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-01-19T00:00:00+00:00'
[2024-02-21T07:59:32.856+0000] {ImapSmtp.py:839} INFO - IMAP search: 'gmail:'Message from RPA Python' after:15-12-2023'
[2024-02-21T07:59:36.039+0000] {logging_mixin.py:188} INFO - Message from RPA Python0
[2024-02-21T07:59:36.040+0000] {ImapSmtp.py:1240} INFO - Attachment filepath: PosixPath('/data/attachments/test-92.txt')
[2024-02-21T07:59:36.041+0000] {ImapSmtp.py:1243} INFO - Saving attachment: test.txt
[2024-02-21T07:59:36.041+0000] {logging_mixin.py:188} INFO - Message from RPA Python1
[2024-02-21T07:59:36.042+0000] {ImapSmtp.py:1240} INFO - Attachment filepath: PosixPath('/data/attachments/test-93.txt')
[2024-02-21T07:59:36.042+0000] {ImapSmtp.py:1243} INFO - Saving attachment: test.txt
[2024-02-21T07:59:36.043+0000] {logging_mixin.py:188} INFO - Message from RPA Python2
[2024-02-21T07:59:36.044+0000] {ImapSmtp.py:1240} INFO - Attachment filepath: PosixPath('/data/attachments/test-94.txt')
[2024-02-21T07:59:36.045+0000] {ImapSmtp.py:1243} INFO - Saving attachment: test.txt
[2024-02-21T07:59:36.045+0000] {logging_mixin.py:188} INFO - Message from RPA Python3
[2024-02-21T07:59:36.046+0000] {ImapSmtp.py:1240} INFO - Attachment filepath: PosixPath('/data/attachments/test-95.txt')
[2024-02-21T07:59:36.047+0000] {ImapSmtp.py:1243} INFO - Saving attachment: test.txt
[2024-02-21T07:59:36.047+0000] {logging_mixin.py:188} INFO - Message from RPA Python4
[2024-02-21T07:59:36.048+0000] {ImapSmtp.py:1240} INFO - Attachment filepath: PosixPath('/data/attachments/test-96.txt')
[2024-02-21T07:59:36.048+0000] {ImapSmtp.py:1243} INFO - Saving attachment: test.txt
[2024-02-21T07:59:36.049+0000] {logging_mixin.py:188} INFO - Message from RPA Python5
[2024-02-21T07:59:36.050+0000] {ImapSmtp.py:1240} INFO - Attachment filepath: PosixPath('/data/attachments/test-97.txt')
[2024-02-21T07:59:36.050+0000] {ImapSmtp.py:1243} INFO - Saving attachment: test.txt
[2024-02-21T07:59:36.050+0000] {logging_mixin.py:188} INFO - Message from RPA Python6
[2024-02-21T07:59:36.051+0000] {ImapSmtp.py:1240} INFO - Attachment filepath: PosixPath('/data/attachments/test-98.txt')
[2024-02-21T07:59:36.051+0000] {ImapSmtp.py:1243} INFO - Saving attachment: test.txt
[2024-02-21T07:59:36.052+0000] {logging_mixin.py:188} INFO - Message from RPA Python7
[2024-02-21T07:59:36.053+0000] {ImapSmtp.py:1240} INFO - Attachment filepath: PosixPath('/data/attachments/test-99.txt')
[2024-02-21T07:59:36.053+0000] {ImapSmtp.py:1243} INFO - Saving attachment: test.txt
[2024-02-21T07:59:36.054+0000] {logging_mixin.py:188} INFO - Message from RPA Python8
[2024-02-21T07:59:36.055+0000] {ImapSmtp.py:1240} INFO - Attachment filepath: PosixPath('/data/attachments/test-100.txt')
[2024-02-21T07:59:36.057+0000] {ImapSmtp.py:1243} INFO - Saving attachment: test.txt
[2024-02-21T07:59:36.058+0000] {logging_mixin.py:188} INFO - Message from RPA Python9
[2024-02-21T07:59:36.062+0000] {ImapSmtp.py:1240} INFO - Attachment filepath: PosixPath('/data/attachments/test-101.txt')
[2024-02-21T07:59:36.063+0000] {ImapSmtp.py:1243} INFO - Saving attachment: test.txt
[2024-02-21T07:59:36.814+0000] {python.py:201} INFO - Done. Returned value was: None
[2024-02-21T07:59:36.839+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=extract_email, task_id=get_mail, execution_date=20240119T000000, start_date=20240221T075929, end_date=20240221T075936
[2024-02-21T07:59:36.909+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-02-21T07:59:36.923+0000] {taskinstance.py:3280} INFO - 0 downstream tasks scheduled from follow-on schedule check
